extension/intents/navigation/navigation.js

```diff
@@ -56,14 +56,16 @@ this.intents.navigation = (function() {
 
   this.intentRunner.registerIntent({
     name: "navigation.translate",
-    description: "Translate the given page to English, using Google Translate",
+    description: "Translate the given page to the chosen language, using Google Translate",
     match: `
-    translate (this |) (page | tab | article | site |) (to english |) (for me |)
+    translate (this |) (page | tab | article | site |) to [language:lang]
     `,
-    examples: ["Translate this page", "test:translate"],
+    examples: ["test:translate this page to Spanish", 
+    "test:translate this page to Dutch"],
     async run(context) {
+      const language = context.slots.language;
```

Comment:
And then this should default to English, i.e., `const language = context.slots.language || "en"`

Reply:
Fixed.